package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Shared functionality for websocket providers",
   "version": "3.0.1",
   "homepage": "https://github.com/feathersjs/socket-commons",
-  "main": "lib/",
+  "main": "lib/index.js",
```

Comment:
Can we change it to `"main": "lib/index",` so that it is consistent with the ones where this is already done?

Reply:
i am against changing that the global convention is add main with extension but thats not a problem i will fork of feathers soon complet and at present i maintaint own forks

Reply:
Makes sense. No need to maintain your own forks.